static/css/syna.css

```diff
@@ -12,6 +12,8 @@
 
 .jumbotron {
   border-radius: 0rem !important;
+  width: 100%;
+  padding-left: 0 !important;
```

Comment:
That removes the centering of the jumbotron and the content.

No changes should be needed for jumbotron itself.